lib/logout.js

```diff
@@ -209,9 +206,9 @@ module.exports.logout = function (options) {
 
       // Logout flow in progress, incoming SAMLResponse from SP. (Could be SP initiated or IdP initiated)
       } else if (req.query.SAMLResponse || req.body.SAMLResponse) {
-        function processLogoutResponse(logoutResponse, transactionId) {
+        function processLogoutResponse(state, transactionId) {
```

Comment:
The logout response only says if the logout succeeded or not. It does not contain a sessionIndex or a nameid. We check that the response was for an existing request (by checking the RelayState), the signature (if signed and configured) and the status.